spec/controllers/teams_controller_spec.rb

```diff
@@ -41,6 +41,13 @@
       end
     end
 
+    context 'past teams' do
+      it 'passing year' do
```

Comment:
Hm, I'm afraid this does not say anything... Aside from the description not really telling the reader what's supposed to happen, the test only tests if the `index` action renders the `index.html` template. One can pass arbitrary params to an `index` action. The test would be the same with:
```ruby
get :index, params: { foo: 'bar'}
```

Given that the `index` action is rather complex, this test should probably better target the actual behavior there.